e2e-tests/.ci/server.generate.sh

```diff
@@ -315,6 +315,8 @@ generate_env_files() {
     case "$SERVER" in
     cloud)
       echo "CYPRESS_serverEdition=Cloud" >>.env.cypress
+      echo "CYPRESS_cwsURL=${CWS_URL}" >> .env.cypress
+      echo "CYPRESS_cwsAPIURL=${CWS_URL}" >> .env.cypress
```

Comment:
Are these the same ?

Reply:
These are forklifted from cypress-ui-automation. I'm actually not 100% sure they are needed.
@saturninoabril  do you know if these may be safely removed? We already have the `CWS_URL` variable with he same values, we can propagate it to the Cypress container if needed in the future.

Reply:
Those are mapped to two separate server configs such as `CloudSettings.CWSURL` and `CloudSettings.CWSAPIURL` -- the former to public-facing URL and the other one as internal API. It actually depends on how the portal is deployed and exposed (ex. for other test [here](https://github.com/mattermost/mattermost/blob/master/e2e-tests/cypress/tests/integration/channels/enterprise/self-hosted/self_hosted_purchase_spec.ts#L18-L19)).

I'd say let's remain as is for flexibility in case anyone (or some case) would require it, although in our test setup are the same.